.github/CHANGELOG.md

```diff
@@ -157,6 +157,10 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
 
 <h3>Improvements</h3>
 
+* Tests for the `Permute` operation are adjusted so that tape expansion happens
+  once.
```

Comment:
I'm not so sure this belongs in 'improvements', as it is a bugfix to the tests suite (and not an improvement in PL directly). Perhaps worth moving to Bug fix, and adding a bit more detail here?

Reply:
Oh, good point! Updated